docs/tools/formatter/configuration-reference.md

```diff
@@ -14,10 +14,38 @@ print-width = 100
 use-tabs = true
 ```
 
+## Presets
+
+Instead of configuring individual settings, you can use a preset that provides a complete set of formatter options optimized for specific coding standards.
+
+```toml
+[formatter]
+preset = "laravel"
+```
+
+Available presets:
+
+- **`"default"`** — PER-CS compliant settings (same as the default configuration)
+- **`"psr-12"`** — Optimized for PSR-12 coding standard
+- **`"laravel"`** — Optimized for Laravel framework conventions
```

Comment:
I apologize for the minor point, but I think it would be clearer to say "Laravel Pint" here. This is because we're borrowing the rules from when Pint is used in Laravel applications, not the conventions of the Laravel framework itself. The laravel/framework itself appears to use an empty preset.
https://github.com/laravel/framework/blob/12.x/pint.json#L2